src/core/AutoRest/Program.cs

```diff
@@ -91,13 +91,6 @@ private static int Main(string[] args)
                             return 0;
                         }
 
-                        // comparison?
```

Comment:
not sure I understand correctly: we haven't moved to `-FANCY` yet (and even then, we may still make the true legacy CLI accessible for a while). I suspect 90% of AutoRest users still run this code without `--server`, so I think we still support such an execution path.

Of course, removing the `-help` path may boost migration a bit.